Javadoc errors

<!--
    This template is for issue report\feature request. 
    If you have any questions, please stop by our chatroom: https://gitter.im/codeborne/selenide
    Note that leaving sections blank will make it difficult for us to troubleshoot and we may have to close the issue.
-->
## The problem
There are lots of JavaDoc warning in the build

## Details
This is preventing documentation that is present in the comments from being displayed in the generated javadocs

## Tell us about your environment
* **Selenide Version:**
Latest
* **Chrome\Firefox\IE Version:**
* **Browser Driver Version:**
* **Selenium Version:**  HEAD master
* **OS Version:**

## Code To Reproduce Issue [ Good To Have ]
`gradle clean javadocs `

produces the following errors 

`20:00:56: Executing task 'javadoc'...

Parallel execution with configuration on demand is an incubating feature.
> Task :libsProd
> Task :libsTest

> Task :compileJava
Note: /home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/SelenideElement.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/testng/SoftAsserts.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :processResources
> Task :classes

> Task :javadoc
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Browsers.java:23: error: unknown tag: dependency
   * <dependency org="org.seleniumhq.selenium" name="selenium-htmlunit-driver" rev="2.+" conf="test->default"/>
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Browsers.java:52: error: malformed HTML
   * <dependency org="com.machinepublishers" name="jbrowserdriver" rev="[0.13.0, 2.0)" conf="test-&gt;default"/&gt;
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:28: warning: no @param for expectedSize
  public static CollectionCondition size(int expectedSize) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:28: warning: no @return
  public static CollectionCondition size(int expectedSize) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:57: warning: no @param for expectedTexts
  public static CollectionCondition texts(String... expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:57: warning: no @return
  public static CollectionCondition texts(String... expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:66: warning: no @param for expectedTexts
  public static CollectionCondition texts(List<String> expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:66: warning: no @return
  public static CollectionCondition texts(List<String> expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:75: warning: no @param for expectedTexts
  public static CollectionCondition textsInAnyOrder(String... expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:75: warning: no @return
  public static CollectionCondition textsInAnyOrder(String... expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:84: warning: no @param for expectedTexts
  public static CollectionCondition textsInAnyOrder(List<String> expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:84: warning: no @return
  public static CollectionCondition textsInAnyOrder(List<String> expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:93: warning: no @param for expectedTexts
  public static CollectionCondition exactTexts(String... expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:93: warning: no @return
  public static CollectionCondition exactTexts(String... expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:102: warning: no @param for expectedTexts
  public static CollectionCondition exactTexts(List<String> expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:102: warning: no @return
  public static CollectionCondition exactTexts(List<String> expectedTexts) {
                                    ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:109: warning: no @param for explanation
  public CollectionCondition because(String explanation) {
                             ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/CollectionCondition.java:109: warning: no @return
  public CollectionCondition because(String explanation) {
                             ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:129: warning: no @return
  public static Condition attribute(final String attributeName, final String expectedAttributeValue) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:156: warning: no @return
  public static Condition value(final String expectedValue) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:175: warning: no @return
  public static Condition exactValue(String value) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:184: warning: no @return
  public static Condition name(String name) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:193: warning: no @return
  public static Condition type(String type) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:202: warning: no @return
  public static Condition id(String id) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:220: warning: no @param for text
  public static Condition matchesText(String text) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:220: warning: no @return
  public static Condition matchesText(String text) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:231: warning: no @return
  public static Condition matchText(final String regex) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:254: warning: no @return
  public static Condition text(final String text) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:265: warning: no @return
  public static Condition textCaseSensitive(final String text) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:287: warning: no @return
  public static Condition exactText(final String text) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:308: warning: no @return
  public static Condition exactTextCaseSensitive(final String text) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:334: warning: no @param for cssClass
  public static Condition cssClass(final String cssClass) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:334: warning: no @return
  public static Condition cssClass(final String cssClass) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:368: warning: no @return
  public static Condition cssValue(final String propertyName, final String expectedValue) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:486: warning: no @param for condition
  public static Condition not(final Condition condition) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:486: warning: no @return
  public static Condition not(final Condition condition) {
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:655: warning: no @param for driver
  public abstract boolean apply(Driver driver, WebElement element);
                          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:666: warning: no description for @param
   * @param driver
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:677: warning: no @param for message
  public Condition because(String message) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Condition.java:677: warning: no @return
  public Condition because(String message) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:65: warning: no @param for expectedSize
  public ElementsCollection shouldHaveSize(int expectedSize) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:65: warning: no @return
  public ElementsCollection shouldHaveSize(int expectedSize) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:72: warning: no @param for conditions
  public ElementsCollection shouldBe(CollectionCondition... conditions) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:72: warning: no @return
  public ElementsCollection shouldBe(CollectionCondition... conditions) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:85: warning: no @param for conditions
  public ElementsCollection shouldHave(CollectionCondition... conditions) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:85: warning: no @return
  public ElementsCollection shouldHave(CollectionCondition... conditions) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:94: warning: no @param for condition
  public ElementsCollection shouldHave(CollectionCondition condition, long timeoutMs) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:94: warning: no @return
  public ElementsCollection shouldHave(CollectionCondition condition, long timeoutMs) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:168: warning: no description for @param
   * @param condition
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:179: warning: no description for @param
   * @param condition
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:189: warning: no description for @param
   * @param condition
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:200: warning: no description for @param
   * @param condition
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:210: warning: no description for @param
   * @param condition
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:221: warning: no description for @param
   * @param condition
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:259: warning: no description for @param
   * @param elements
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:262: warning: no @param for driver
  public static String elementsToString(Driver driver, Collection<WebElement> elements) {
                       ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:288: warning: no description for @return
   * @return
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:299: warning: no description for @return
   * @return
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:308: warning: no description for @return
   * @return
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:319: warning: no @return
  public ElementsCollection first(int elements) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:328: warning: no @return
  public ElementsCollection last(int elements) {
                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/ElementsCollection.java:335: warning: no description for @return
   * @return
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/FileDownloadMode.java:6: error: attribute lacks value
   * Works only for <a href></a> elements.
                       ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:24: warning: no @param for fileName
  public static String takeScreenShot(String fileName) {
                       ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:40: warning: no @param for element
  public static File takeScreenShot(WebElement element) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:49: warning: no @param for iframe
  public static File takeScreenShot(WebElement iframe, WebElement element) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:49: warning: no @param for element
  public static File takeScreenShot(WebElement iframe, WebElement element) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:58: warning: no @param for iframe
  public static BufferedImage takeScreenShotAsImage(WebElement iframe, WebElement element) {
                              ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:58: warning: no @param for element
  public static BufferedImage takeScreenShotAsImage(WebElement iframe, WebElement element) {
                              ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Screenshots.java:66: warning: no @param for element
  public static BufferedImage takeScreenShotAsImage(WebElement element) {
                              ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:39: error: unknown attribute: binding
   * <div binding="fieldValue"></div>
          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:44: error: unknown attribute: binding
   * <div binding="userName17fk5n6kc2Ds45F40d0fieldValue_promoLanding word"></div>
          ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:58: error: malformed HTML
   * Seems to work incorrectly if attribute name contains dash, for example: <option data-mailServerId="123"></option>
                                                                             ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:58: error: bad use of '>'
   * Seems to work incorrectly if attribute name contains dash, for example: <option data-mailServerId="123"></option>
                                                                                                            ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:58: error: unknown tag: option
   * Seems to work incorrectly if attribute name contains dash, for example: <option data-mailServerId="123"></option>
                                                                                                             ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:73: warning: no @param for attributeName
  public static By by(String attributeName, String attributeValue) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:73: warning: no @param for attributeValue
  public static By by(String attributeName, String attributeValue) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:73: warning: no @return
  public static By by(String attributeName, String attributeValue) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:80: warning: no @param for title
  public static By byTitle(String title) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:80: warning: no @return
  public static By byTitle(String title) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:87: warning: no @param for value
  public static By byValue(String value) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:87: warning: no @return
  public static By byValue(String value) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:130: warning: no @param for name
  public static By byName(String name) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:130: warning: no @return
  public static By byName(String name) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:138: warning: no @param for xpath
  public static By byXpath(String xpath) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:138: warning: no @return
  public static By byXpath(String xpath) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:146: warning: no @param for linkText
  public static By byLinkText(String linkText) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:146: warning: no @return
  public static By byLinkText(String linkText) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:154: warning: no @param for partialLinkText
  public static By byPartialLinkText(String partialLinkText) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:154: warning: no @return
  public static By byPartialLinkText(String partialLinkText) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:162: warning: no @param for id
  public static By byId(String id) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:162: warning: no @return
  public static By byId(String id) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:170: warning: no @param for css
  public static By byCssSelector(String css) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:170: warning: no @return
  public static By byCssSelector(String css) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:178: warning: no @param for className
  public static By byClassName(String className) {
                   ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/Selectors.java:178: warning: no @return
  public static By byClassName(String className) {
                   ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:44: warning: no @param for absoluteUrl
  public static void open(URL absoluteUrl) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:61: warning: no @param for relativeOrAbsoluteUrl
  public static void open(String relativeOrAbsoluteUrl, String domain, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:61: warning: no @param for domain
  public static void open(String relativeOrAbsoluteUrl, String domain, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:61: warning: no @param for login
  public static void open(String relativeOrAbsoluteUrl, String domain, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:61: warning: no @param for password
  public static void open(String relativeOrAbsoluteUrl, String domain, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:77: warning: no @param for relativeOrAbsoluteUrl
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:77: warning: no @param for authenticationType
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:77: warning: no @param for login
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:77: warning: no @param for password
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, String login, String password) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:95: warning: no @param for relativeOrAbsoluteUrl
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, Credentials credentials) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:95: warning: no @param for authenticationType
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, Credentials credentials) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:95: warning: no @param for credentials
  public static void open(String relativeOrAbsoluteUrl, AuthenticationType authenticationType, Credentials credentials) {
                     ^
/home/rkielty/devel/gh/selenide/statics/src/main/java/com/codeborne/selenide/Selenide.java:635: error: unexpected text
   * @see org.openqa.selenium.logging.LogType,
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/SelenideElement.java:56: error: unexpected end tag: </p>
   * </p>
     ^
/home/rkielty/devel/gh/selenide/src/main/java/com/codeborne/selenide/WebDriverProvider.java:19: error: unexpected end tag: </p>
 * </p>
   ^
11 errors
100 warnings

BUILD SUCCESSFUL in 7s
5 actionable tasks: 5 executed
20:01:03: Task execution finished 'javadoc'.
`

--- a/src/main/java/com/codeborne/selenide/Browsers.java
+++ b/src/main/java/com/codeborne/selenide/Browsers.java
@@ -45,7 +45,7 @@ public interface Browsers {
 
   /**
    * To use JbrowserDriver, you need to include extra dependency to your project:
-   * <dependency org="com.machinepublishers" name="jbrowserdriver" rev="[0.13.0, 2.0)" conf="test-&gt;default"/&gt;
+   * {@code <dependency org="com.machinepublishers" name="jbrowserdriver" rev="[0.13.0, 2.0)" conf="test-&gt;default"/&gt;}
    */
   public static final String JBROWSER = "jbrowser";
 }

--- a/src/main/java/com/codeborne/selenide/FileDownloadMode.java
+++ b/src/main/java/com/codeborne/selenide/FileDownloadMode.java
@@ -3,7 +3,7 @@ package com.codeborne.selenide;
 public enum FileDownloadMode {
   /**
    * Download files via direct http request.
-   * Works only for <a href></a> elements.
+   * Works only for {@code <a href></a>} elements.
    * Sends GET request to "href" with all cookies from current browser session.
    */
   HTTPGET,

--- a/src/main/java/com/codeborne/selenide/Selectors.java
+++ b/src/main/java/com/codeborne/selenide/Selectors.java
@@ -36,12 +36,12 @@ public class Selectors {
    * Find elements having attribute with given value.
    *
    * Examples:
-   * <div binding="fieldValue"></div>
+   * {@code <div binding="fieldValue"></div>}
    * Find element with attribute 'binding' EXACTLY containing text 'fieldValue' , use:
    * byAttribute("binding", "fieldValue")
    *
    * For finding difficult/generated data attribute which contains some value:
-   * <div binding="userName17fk5n6kc2Ds45F40d0fieldValue_promoLanding word"></div>
+   * {@code <div binding="userName17fk5n6kc2Ds45F40d0fieldValue_promoLanding word"></div>}
    *
    * Find element with attribute 'binding' CONTAINING text 'fieldValue', use symbol '*' with attribute name:
    * byAttribute("binding*", "fieldValue") it same as By.cssSelector("[binding*='fieldValue']")
@@ -55,7 +55,7 @@ public class Selectors {
    * Find element whose attribute 'binding' CONTAINING WORD 'word':
    * byAttribute("binding~", "word")
    *
-   * Seems to work incorrectly if attribute name contains dash, for example: <option data-mailServerId="123"></option>
+   * Seems to work incorrectly if attribute name contains dash, for example: {@code <option data-mailServerId="123"></option>}
    *
    * @param attributeName name of attribute, should not be empty or null
    * @param attributeValue value of attribute, should not contain both apostrophes and quotes

--- a/src/main/java/com/codeborne/selenide/SelenideElement.java
+++ b/src/main/java/com/codeborne/selenide/SelenideElement.java
@@ -46,14 +46,14 @@ public interface SelenideElement extends WebElement, WrapsDriver, WrapsElement, 
   SelenideElement val(String text);
 
   /**
-   * <p>Append given test to the text field and trigger "change" event.</p>
-   *
+   * Append given test to the text field and trigger "change" event.
    * <p>
    * Implementation details:
-   * This is the same as <pre>
+   * This is the same as
+   * <pre>
    *   1. WebElement.sendKeys(text)
-   *   2. Trigger change event</pre>
-   * </p>
+   *   2. Trigger change event
+   * </pre>
    *
    * @param text Any text to append into the text field.
    *

--- a/src/main/java/com/codeborne/selenide/WebDriverProvider.java
+++ b/src/main/java/com/codeborne/selenide/WebDriverProvider.java
@@ -16,7 +16,6 @@ import org.openqa.selenium.remote.DesiredCapabilities;
  * constructor and pass this class name as {@code browser} system variable value.
  * <li>Implement this very interface and pass the implementation class name as {@code browser} system variable value.
  * </ul>
- * </p>
  */
 public interface WebDriverProvider {
 

--- a/statics/src/main/java/com/codeborne/selenide/Selenide.java
+++ b/statics/src/main/java/com/codeborne/selenide/Selenide.java
@@ -4,6 +4,7 @@ import com.codeborne.selenide.ex.DialogTextMismatch;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.logging.LogType;
 
 import java.io.File;
 import java.io.IOException;
@@ -632,8 +633,8 @@ public class Selenide {
    * @param logType WebDriver supported log types
    * @param logLevel logging level that will be used to control logging output
    * @return list of log entries
-   * @see org.openqa.selenium.logging.LogType,
-   * @see java.util.logging.Level
+   * @see LogType
+   * @see Level
    */
   public static List<String> getWebDriverLogs(String logType, Level logLevel) {
     return getSelenideDriver().getWebDriverLogs().logs(logType, logLevel);

